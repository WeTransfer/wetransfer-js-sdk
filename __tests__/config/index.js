const config = require('../../src/config');

describe('Default configuration', () => {
  it('should match defined configuration', () => {
    expect(config).toMatchSnapshot();
  });
});
