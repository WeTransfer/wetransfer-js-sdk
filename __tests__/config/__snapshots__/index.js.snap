// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default configuration should match defined configuration 1`] = `
Object {
  "chunkRetries": 30,
  "chunkRetryDelay": 1000,
  "concurrency": 5,
  "logger": Object {
    "level": "info",
  },
  "retries": 15,
  "retryDelay": [Function],
}
`;
